### DISCUSSION TITLE
remix

### DISCUSSION THREAD

**Discussion by imtiaz70** [nehal2210/Blockchain-Engineer-Course#9] [General]
remix only shows the **public function's** button on the left panel, it doesn't show a button for **internal functions**, how can the internal function's button be shown?

**Comment by nehal2210** [comment]
we make the variable internal because we don't want to show it . But if you want to read via button, you can create a public function and inside this public function, you can return an internal variable.

**Reply by Muhammad-Ali-bhatti** [reply]
But my Public Variable Button Is Missing Bhai What Should I do

**Reply by nehal2210** [reply]
How's that possible

**Reply by nehal2210** [reply]
connect with me if did not resolved the issue
https://www.linkedin.com/in/nehal-ahmed-072949193/

**Reply by Muhammad-Ali-bhatti** [reply]
> How's that possible

It's okay thank you so much
